assets/sip_samples/2D_fa307608-35c3-11ed-9243-7e92631d7d27/data/representations/representation_1/metadata/preservation/premis.xml

```diff
@@ -23,8 +23,8 @@
       <premis:relationshipType authority="relationshipType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipType/str">structural</premis:relationshipType>
       <premis:relationshipSubType authority="relationshipSubType" authorityURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType" valueURI="http://id.loc.gov/vocabulary/preservation/relationshipSubType/rep">represents</premis:relationshipSubType>
       <premis:relatedObjectIdentifier>
-        <premis:relatedObjectIdentifierType>VIAA PID</premis:relatedObjectIdentifierType>
```

Comment:
bij nader inzien: 
- `VIAA PID` wordt `MEEMOO-PID`
- `VIAA SP Agent ID` wordt `MEEMOO-OR-ID`
- `local_id` wordt `MEEMOO-LOCAL-ID`

Reply:
@spacid done in https://github.com/viaacode/documentation/pull/90/commits/4819015d7f573a615ec394c4e688a07d66066762 Check jij?